### PR TITLE
theming

### PR DESCRIPTION
**Pull request by speedcuber911**

### CODE REVIEW

File: src/components/mainPage/MainAppPage.js [hunk 1]
```diff
@@ -28,8 +28,8 @@ export function MainAppPage() {
     <ThemeContext.Provider value={theme}>
```
Comment (Dheeraj-parashar) [inline src/components/mainPage/MainAppPage.js:28]: format the code

File: src/components/mainPage/MainAppPage.js [hunk 2]
```diff
@@ -28,8 +28,8 @@ export function MainAppPage() {
...
       <UserContext.Provider value={user}>
         <div>
-          <Navbar />
```
Comment (syantanroy) [inline src/components/mainPage/MainAppPage.js:31]: Remove the space sir

Comment (speedcuber911) [reply src/components/mainPage/MainAppPage.js:31]: Sure

**Review by rajdhaman — APPROVED** [review]
approved

**Pull request merged**